jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/container/TabSheetLoader.java

```diff
@@ -53,11 +56,19 @@ public void loadComponent() {
     protected void createTabs(Element element) {
         LayoutLoader loader = getLayoutLoader();
 
+        boolean firstTab = true;
         for (Element subElement : element.elements("tab")) {
-            ComponentLoader<?> componentLoader = loader.getLoader(subElement, TabLoader.class);
-            componentLoader.initComponent();
+            boolean shouldBeLazy = loadBoolean(subElement, "lazy").orElse(false) && !firstTab;
+            ComponentLoader<?> componentLoader = loader.getLoader(subElement,
+                    shouldBeLazy ? LazyTabLoader.class : TabLoader.class);
 
+            componentLoader.initComponent();
+            if (shouldBeLazy) {
+                resultComponent.addSelectedChangeListener(((LazyTabLoader) componentLoader)::selectedChangeHandler);
```

Comment:
1. You still create a separate `SelectedChangeListener` for each lazy tab
2. I doubt that `LazyTabLoader` should be used as `SelectedChangeListener`